packages/features/calendars/weeklyview/state/store.ts

```diff
@@ -32,9 +32,7 @@ export const useCalendarStore = create<CalendarStoreProps>((set) => ({
     let events = state.events;
 
     if (state.sortEvents) {
-      events = state.events.sort(
-        (a, b) => dayjs(a.start).get("milliseconds") - dayjs(b.start).get("milliseconds")
-      );
+      events = state.events.sort((a, b) => dayjs(a.start).valueOf() - dayjs(b.start).valueOf());
```

Comment:
I spent 3 hours wondering why my maths wasnt working for overlap (this sort  wasnt sorting - that deffo didnt help)